modifiers/ReplaceBlockPairs.groovy

```diff
@@ -2,16 +2,15 @@ import net.querz.mcaselector.io.mca.ChunkData
 import net.querz.nbt.CompoundTag
 
 /**
- * Defines pairs of blocks to replace.
+ * Specify blocks as comma-separated pairs in the format 'source:original_block': 'source:new_block'
  * Replaces the original block in each block pair with its respective destination block.
- * Note: Block states are not updated, only the block name is changed.
  *
+ * Note: Block states are not updated, only the block name is changed.
```

Comment:
move this back up again so that the annotations (`@...`) are standing separately